projects.md

```diff
@@ -0,0 +1,60 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
```

Comment:
Not a blocker, but maybe a spin-out issue: Only bizdev has access, so this recommendation isn't clearly actionable atm. Potential resolutions:
- no user limit, so could just invite all members as reg (non-admin) users
- create a shared user and offer credentials
- direct people to quickly add things to `hyphacoop/organizing-private`, and someone on bizdev will worry about updating CRM
- change copy to indicate that bizdev will handle leads, just pass em on

https://knowledge.hubspot.com/settings/hubspot-user-permissions-guide

happy to punt this into organizing#147

Reply:
That's a logistical problem. If someone wants to enter a lead they will inquiry best way from WG. If they just want to announce a grant someone can put in for them, but if it's a client they have follow ups it may be better to make an account. Either way, I don't think that logistical issue needs to be resolved here in this doc.

Reply:
```suggestion
Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
Ask Business Planning WG for help doing this. 
```

What about this?